### PR TITLE
Add post about running Kibana with Sense plugin locally

### PR DESCRIPTION
**Pull request by tonysm**
### Added

- Add blogpost about running Kibana with Sense plugin locally.

### CODE REVIEW

File: _posts/2017-12-18-running-kibana-with-sense.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: "Running Kibana with the Sense plugin"
+date: 2017-12-18
+authors: ["tony"]
+categories: [docker,kibana,elasticsearch,sense]
+excerpt: I used to debug some Elasticsearch queries using the Sense Chrome extension, but since it got blocked on Chrome due to malware I have been missing it. But Kibana has the Sense plugin on it since version 4, let's see how we can use it in our local docker setup.
+---
+
+Lately we have been missing the Sense Chrome extension, basically because of the autocompletes when we are writing some Elasticsearch queries or debugging something. [It got blocked on Chrome due to a malware](https://www.elastic.co/blog/sense-chrome-plugin-malware-issue), and it was when I discored it was actually not maintained by Elastic.
```
Comment (pedro-santiago) [inline _posts/2017-12-18-running-kibana-with-sense.md]: discovered

File: _posts/2017-12-18-running-kibana-with-sense.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+In the post, they mention Kibana has a console with the same funcionality that we were missing from Sense extension, so I decided to give it a try.
```
Comment (pedro-santiago) [inline _posts/2017-12-18-running-kibana-with-sense.md]: functionality

File: _posts/2017-12-18-running-kibana-with-sense.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+Before we jump in, let's briefily discuss our setup for the project. We use:
+
+- Docker for local development
+- Elasticsearch 2.4
+
+We haven't had time to upgrade Elasticsearch yet, and I thought it was going to be a problem, but it wasn't. Kibana has the console since version 4, and that version works fine with Elasticsearch 2.*.
+
+## Setting up
+
```
Comment (pedro-santiago) [inline _posts/2017-12-18-running-kibana-with-sense.md]: available

File: _posts/2017-12-18-running-kibana-with-sense.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+Before we jump in, let's briefly discuss our setup for the project. We use:
+
+- Docker for local development
+- Elasticsearch 2.4
+
+We haven't had time to upgrade Elasticsearch yet, and I thought it was going to be a problem, but it wasn't. Kibana has the console since version 4, and that version works fine with Elasticsearch 2.*.
+
+## Setting up
+
+As I said, we use Docker locally. But the sense Kibana plugin is not available in the official docker image in a pre-baked way, so I had to create our own custom image with it. Create a file under `resources/docker/kibana/Dockerfile` with the content:
```
Comment (dieterve) [inline _posts/2017-12-18-running-kibana-with-sense.md]: Capital `S` ense?

**Pull request merged**